tests/NHSD.BuyingCatalogue.Identity.Api.IntegrationTests/Features/Disable User.3543.feature

```diff
@@ -0,0 +1,56 @@
+﻿Feature: Disable User
+	As an Authority User
+	I want to be able to disable Users accounts
+	So that the user would not be able to access the Buying Catalogue
+
+Background:
+	Given Organisations exist
+		| Name           | OdsCode |
+		| Organisation 1 | Ods 1   |
+	And Users exist
+		| Id  | OrganisationName | FirstName | LastName | Email             | PhoneNumber | Disabled | Password | OrganisationFunction |
+		| 123 | Organisation 1   | John      | Doe      | authority@doe.com | 01234567890 | false    | yolo     | Authority            |
+		| 234 | Organisation 1   | Jane      | Doe      | buyer@doe.com     | 01234567890 | false    | oloy     | Buyer                |
+
+@3543
+Scenario: 1. An authority user can disable a users account
+	Given a user is logged in
+		| Username          | Password | Scope        |
+		| authority@doe.com | yolo     | Organisation |
+	When a POST request is made to disable user 234
+	Then a response with status code 204 is returned
+	When a GET request is made for a user with id 234
```

Comment:
Rather than sending a GET user request, could the code just go to the database? As it stands the GET user request is coupled with a lot more tests, so if there is a change to that endpoint which causes the test to fail, there will be a lot more tests failing, rather than hopefully just one.